### DISCUSSION TITLE
Quickstart Howto for Using Flowershow as website template

### DISCUSSION THREAD

**Discussion by rufuspollock** [flowershow/flowershow#670] [General and Help]
From @olayway in a thread on discord 

1. clone the repo
2. add any of your markdown files to /content folder
3. run npm run dev or npm run export && npm run start
4. (optional) configure your site with /content/config.mjs file (see available options here: https://flowershow.app/docs/config and in feature specific docs pages https://flowershow.app/docs)
5. (optional) If you want to embed assets, you need to put them in some folder iside the /content folder (e.g. /content/assets) and create a symlink to that folder in /public under the same name (here assets)

NB: we have this old (may be a bit out of date) tutorial https://flowershow.app/blog/2023-02-16-nextjs-tutorial

## Acceptance

- [ ] Instructions for how to do this
- [ ] Tried out
- [ ] Documented somewhere good (e.g. `docs/website-quickstart`?)

**Comment by davidgasquez** [comment]
Would be cool to run Flowershow from `npm` a la Astro/Eleventy/Evidence (`npm create astro/eleventy/evidence@latest && npm run dev`). Smooth way to get something up without messing with `git`!

EDIT: The command I'm looking for might be [already in the docs](https://flowershow.app/blog/2023-02-16-nextjs-tutorial)!

```
npx create-next-app@latest --example https://github.com/datopian/flowershow
```

**Comment by rufuspollock** [comment]
@davidgasquez yes i think that command should "just work". So maybe we just need to document that really clearly ...
